go/vt/mysqlctl/mysql_daemon_test.go

```diff
@@ -0,0 +1,49 @@
+// Copyright 2015, Google Inc. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package mysqlctl
+
+import (
+	"reflect"
+	"testing"
+
+	myproto "github.com/youtube/vitess/go/vt/mysqlctl/proto"
+)
+
+// Test that the fake mysql .GetSchema() implementation respects the excludeTables parameter.
+func TestFakeMysqlDaemonGetSchemaExcludeTables(t *testing.T) {
+	fmd := FakeMysqlDaemon{}
+
+	includedTable := &myproto.TableDefinition{
+		Name:              "includedTable",
+		Columns:           []string{"id", "msg", "keyspace_id"},
+		PrimaryKeyColumns: []string{"id"},
+		Type:              myproto.TABLE_BASE_TABLE,
+	}
+	excludedTable := &myproto.TableDefinition{
+		Name:              "excludedTable",
+		Columns:           []string{"id", "msg", "keyspace_id"},
+		PrimaryKeyColumns: []string{"id"},
+		Type:              myproto.TABLE_BASE_TABLE,
+	}
+
+	fmd.Schema = &myproto.SchemaDefinition{
+		DatabaseSchema: "",
+		TableDefinitions: []*myproto.TableDefinition{
+			includedTable,
+			excludedTable,
+		},
+	}
+
+	schema, error := fmd.GetSchema("dbName", nil /* tables */, []string{excludedTable.Name}, true /* includeViews */)
+	if error != nil {
+		t.Fatal("GetSchema must not return an error")
+	}
+	if len(schema.TableDefinitions) != 1 {
+		t.Fatal("there should be only one included table")
```

Comment:
I don't think it matters too much in this particular case, but idiomatic Go is to generally do a t.Errof instead of Fatalf, unless there is no meaningful information that can be gathered by running the test until the end...

This seems like more of  an error case than a fatal case. For example, it could be useful to know that there were multiple returned schema, but the expected one is equal.